### DISCUSSION TITLE
Cached views and references to sensitive data in pull requests not removed by GitHub Support

### DISCUSSION THREAD

**Discussion by oliviadodge** [community/community#47294] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I recently had to remove some sensitive data from a private repo I'm working on, following the instructions in https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository
The last step was to contact GitHub Support to ask that cached views and references to sensitive data in the pull requests be removed. The removal was confirmed by GitHub, but I am still seeing the hanging commits and PR's that reference that data. 

Does it take a while for the process to be complete even after confirmation from GitHub, or should it be instantaneous?

This is blocking us from making the repo public.

Thanks in advance!

**Comment by mesuhailpm** [comment]
How did you reach out to them?
